src/Stack/Constants.hs

```diff
@@ -254,34 +254,30 @@ wiredInPackages = case mparsed of
   Nothing -> impureThrow WiredInPackagesNotParsedBug
  where
   mparsed = mapM parsePackageName
-    [ "ghc-prim"
+    [ "rts"
+      -- Said to be not a \'real\' package
+    , "ghc-prim"
+      -- A magic package
+    , "ghc-bignum"
       -- A magic package
     , "integer-gmp"
-      -- No longer magic > 1.0.3.0. With GHC 9.6.4 at least, there seems to be
-      -- no problem in using it.
+      -- No longer magic > 1.0.3.0 (GHC >= 9.0) and deprecated in favour of
+      -- ghc-bignum. With GHC 9.6.4 at least, there seems to be no problem in
+      -- using it.
     , "integer-simple"
       -- A magic package
     , "base"
       -- A magic package
-    , "rts"
-      -- Said to be not a \'real\' package
     , "template-haskell"
       -- A magic package
-    , "dph-seq"
-      -- Deprecated in favour of dph-prim-seq, which does not appear to be
-      -- magic. With GHC 9.6.4 at least, there seems to be no problem in using
-      -- it.
-    , "dph-par"
-      --  Deprecated in favour of dph-prim-par, which does not appear to be
-      -- magic. With GHC 9.6.4 at least, there seems to be no problem in using
-      -- it.
     , "ghc"
       -- A magic package
     , "interactive"
```

Comment:
The comment below seems to indicate that interactive could be removed, but it hasn't been. Was that intentional?

Reply:
@chreekat, do you mean the comment that starts "Type and class declarations at the GHCi command prompt"? I think the relationship between `interactive` and the versions of GHC that Stack currently supports (GHC >= 8.4) has not changed.

Reply:
I think I misread the comment. Disregard.